assets/scss/sidebar.scss

```diff
@@ -0,0 +1,19 @@
+.b-sidebar-backdrop.bg-dark {
+  background-color: rgb(0, 0, 0) !important;
+  opacity: 0.7;
+}
+
+
+#sidebar {
+  width: 16rem;
+  max-height: 100vh;
+  transition: $standard-transition; // fixes header appear/disappear
+}
+// fixes header appear/disappear
+.container-fluid:not(.show) {
+  #sidebar, .b-sidebar-backdrop {
+    transform: translateY(3.5rem);
+    transition: $standard-transition;
+  }
+}
```

Comment:
Not sure if it is because of any of the styling here, but the sidebar has a horizontal scrollbar on larger screens. I have tested this in Chrome and Firefox. Mobile is fine, but desktop sizes shows a scrollbar.

Reply:
Well spotted !